bin/dev/create_helm_charts_pr.sh

```diff
@@ -53,6 +57,25 @@ rm -rf stable/uaa
 cp -r ../bundle/helm/cf stable/
 cp -r ../bundle/helm/uaa stable/
 
+# Show the places in the charts having RC references
+grep -rn '.-rc' stable/{cf,uaa}
+
+# Fixing RC references.
+
+# CLUSTER_BUILD: "2.20.2-rc1"
+sed -i 's|-rc.*"$|"|' stable/cf/values.yaml
+sed -i 's|-rc.*"$|"|' stable/uaa/values.yaml
+
+# scfVersion: 2.20.2-rc1+cf12.17.0.0.g5c8dc458
+sed -i 's|-rc.*cf|+cf|' stable/cf/Chart.yaml
+sed -i 's|-rc.*cf|+cf|' stable/uaa/Chart.yaml
```

Comment:
Might be good to comment that this is _not_ a typo, we do indeed say `cf` in the UAA chart.
Though this might still be better as:
```suggestion
sed -i 's|-rc[0-9]\+||' stable/uaa/Chart.yaml
```

Reply:
Will trial this without `cf`. Note, will also go to multi-digit. As much as we wish to keep the number of RCs down ...

Reply:
Addressed in commit a38786f3eb1a4343fe7bb3a205775774729fa753.